js/tawk_to.admin.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+// initialize variables
+var tawktoSettings = {};
+
+jQuery(document).ready(function() {
+  tawktoSettings = Drupal.settings.tawkto || {};
+
+  var currentHost = window.location.protocol + "//" + window.location.host;
+  var url = tawktoSettings.iframeUrl + "&pltf=drupal&parentDomain=" + currentHost;
```

Comment:
I think platform should be included in the settings URL already (from helper)

Reply:
It's not there currently but it does make more sense to have in there. I'll move it